### DISCUSSION TITLE
Unsure how portRange works?

### DISCUSSION THREAD

**Discussion by regnaio** [geckosio/geckos.io#224] [Help / Questions]
I see in the docs:

Allows you to set a custom port range for the WebRTC connection
```js
// server.js
const io = geckos({
  portRange: {
    min: 10000,
    max: 20000
  }
})
```

But in chrome://webrtc-internals/, I see icecandidates with ports with values around ~59000

How can we verify that `portRange` is indeed limiting the port range?

Thanks!

**Comment by yandeu** [comment]
Not sure how it works on localhost, but if you deploy it on a server and only open ports x to y, you should be able to verify that it works.
